packages/vsce/src/SshConfigUtils.ts

```diff
@@ -34,145 +39,262 @@ export class SshConfigUtils {
         const qpItems: vscode.QuickPickItem[] = [
             { label: "$(plus) Add New SSH Host..." },
-            ...sshProfiles.map((prof) => ({ label: prof.name!, description: prof.profile?.host })),
-            { label: "Migrate From SSH Config", kind: vscode.QuickPickItemKind.Separator },
-            ...migratedConfigs.map((config) => ({ label: config.name!, description: config.hostname })),
+            ...sshProfiles.map(({ name, profile }) => ({
+                label: name!,
+                description: profile?.host,
+            })),
+            {
+                label: "Migrate From SSH Config",
+                kind: vscode.QuickPickItemKind.Separator,
+            },
+
+            ...filteredMigratedConfigs.map(({ name, hostname }) => ({
+                label: name!,
+                description: hostname,
+            })),
         ];
-        const result = await Gui.showQuickPick(qpItems, { title: "Choose an SSH host" });
-
-        let selectedProfile: sshConfigExt | undefined;
-
-        if (result === qpItems[0] || migratedConfigs.some((config) => result?.label === config.name!)) {
-            // Create new config
-            if (result === qpItems[0]) {
-                let newConfig = await SshConfigUtils.createNewConfig();
-                if (newConfig) {
-                    await profCache.refresh();
-                    if (newConfig && !newConfig.name) {
-                        newConfig = await SshConfigUtils.getNewProfileName(newConfig);
-                        if (!newConfig?.name) {
-                            vscode.window.showWarningMessage("SSH setup cancelled.");
-                            return;
+
+        const result = await Gui.showQuickPick(qpItems, {
+            title: "Choose an SSH host",
+        });
+
+        // If nothing selected, return
+        if (!result) return;
+
+        if (result === qpItems[0] || filteredMigratedConfigs.some(({ name }) => result.label === name)) {
+            const isNewConfig = result === qpItems[0];
+
+            // If result is add new SSH host then create a new config, if not use migrated configs
+            let selectedProfile = isNewConfig
+                ? await SshConfigUtils.createNewConfig()
+                : filteredMigratedConfigs.find(({ name }) => result.label === name);
+
+            if (!selectedProfile) return;
+
+            const workspaceDir = ZoweVsCodeExtension.workspaceRoot;
+
+            // Prioritize creating a team config in the local workspace if it exists even if a global config exists
+            if (
+                workspaceDir?.uri.fsPath !== undefined &&
+                !profInfo.getTeamConfig().layerExists(workspaceDir!.uri.fsPath)
+            ) {
+                await SshConfigUtils.createZoweSchema(false);
+            } else {
+                await SshConfigUtils.createZoweSchema(true);
+            }
```

Comment:
Should we add some TODOs around this block of code since this is something that we want to do only during the POC phase?